integration/spark/app/src/main/java/io/openlineage/spark/agent/lifecycle/RddExecutionContext.java

```diff
@@ -197,17 +199,20 @@ static String nameRDD(RDD<?> rdd) {
   @Override
   public void start(SparkListenerSQLExecutionStart sqlStart) {
     // do nothing
+    log.debug("start SparkListenerSQLExecutionStart {}", sqlStart);
   }
 
   @Override
   public void end(SparkListenerSQLExecutionEnd sqlEnd) {
     // do nothing
+    log.debug("start SparkListenerSQLExecutionEnd {}", sqlEnd);
   }
 
   @Override
   public void start(SparkListenerJobStart jobStart) {
-    if (inputs.isEmpty() && outputs.isEmpty()) {
-      log.info("RDDs are empty: skipping sending OpenLineage event");
+    log.debug("start SparkListenerJobStart {}", jobStart);
+    if (outputs.isEmpty()) {
+      log.info("Output RDDs are empty: skipping sending OpenLineage event");
```

Comment:
Is this a case where we would want to track that we don't understand the output or is it a case where we don't want to send an event? If the latter, please add a comment that explains this.

Reply:
In case we don't understand the output, we use `log.warn` to notify it. However, there are cases when it does not make sense to send OL event as the event will not add any value to a lineage graph. This can be Spark action like `collect`/`count` or some internal trigger on `OpenlineageSparkListener`.